src/app/component/SwipeableCartItem/SwipeableCartItem.container.js

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types';
+import { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { CartItemType } from 'Type/MiniCart';
+import { makeCancelable } from 'Util/Promise';
+
+import SwipeableCartItem from './SwipeableCartItem.component';
+
+const CartDispatcher = import(/* webpackMode: "lazy", webpackChunkName: "dispatchers" */'Store/Cart/Cart.dispatcher');
+
+export const mapDispatchToProps = (dispatch) => ({
+    removeProduct: (options) => CartDispatcher.then(
+        ({ default: dispatcher }) => dispatcher.removeProductFromCart(dispatch, options)
+    )
+});
+
+export class SwipeableCartItemContainer extends Component {
+    static propTypes = {
+        removeProduct: PropTypes.func.isRequired,
+        item: CartItemType.isRequired
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.handleRemoveCartItem = this.handleRemoveCartItem.bind(this);
+        this.setStateNotLoading = this.setStateNotLoading.bind(this);
+
+        this.handlers = [];
+
+        this.state = { isLoading: false };
```

Comment:
I think you do not need a constructor declaration for that, can simply use proposal-class-properties :)